src/python/pants/backend/go/goals/test.py

```diff
@@ -104,6 +105,12 @@ def opt_out(cls, tgt: Target) -> bool:
         return tgt.get(SkipGoTestsField).value
 
 
+class GoTestRequest(TestRequest):
+    # TODO: Remove the type-ignore after adding a `skip` option to the subsystem.
```

Comment:
(Repeating from PR description): Here and elsewhere I've left these TODOs because I didn't want to mix a plugin API change and a user-facing API change in the same PR. I can go ahead and do these if it's not a big deal to keep them separate - otherwise I'll file an issue in the 2.15.x milestone.